src/chrome/content/rules/beeline.kz.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+beeline.kz/: Redirect for 'https://beeline.kz/' missing Location
+app.beeline.kz non-2xx http code
+nomer.beeline.kz mismatch
+podarok.beeline.kz self signed
+like.beeline.kz timed out
+video.beeline.kz timed out
+love.beeline.kz timed out
+music.beeline.kz mismatch
+fitness.beeline.kz timed out
+kids.beeline.kz mismatch
+football.beeline.kz timed out
+wap.beeline.kz timed out
+book.beeline.kz timed out
+m.beeline.kz timed out
+777.beeline.kz self signed
+-->
+<ruleset name="beeline.kz">
+	<target host="beeline.kz" />
+	<target host="www.beeline.kz" />
```

Comment:
This will cause a redirect loop: https://www.beeline.kz/almatinskaya-obl/customers/mobile redirects to http://www.beeline.kz/almatinskaya-obl/customers/mobile